edit/buildozer.go

```diff
@@ -417,6 +417,46 @@ func cmdRemove(opts *Options, env CmdEnvironment) (*build.File, error) {
 	return nil, nil
 }
 
+func cmdRemoveIfEqual(opts *Options, env CmdEnvironment) (*build.File, error) {
+	attr := env.Args[0]
+	args := env.Args[1:]
+	expr := getAttrValueExpr(attr, args, env)
+	cmp, err := func() (func(build.Expr) bool, error) {
+		switch input := expr.(type) {
+		case *build.StringExpr:
+			return func(attr build.Expr) bool {
+				if val, ok := attr.(*build.StringExpr); ok {
+					return labels.Equal(val.Value, input.Value, env.Pkg)
+				}
+				return false
+			}, nil
+		case *build.Ident:
+			return func(attr build.Expr) bool {
+				switch val := attr.(type) {
+				case *build.StringExpr:
+					return labels.Equal(val.Value, input.Name, env.Pkg)
+				case *build.Ident:
+					return val.Name == input.Name
+				default:
+					return false
+				}
+			}, nil
+		default:
+			return nil, fmt.Errorf("unsupported expression type %T", input)
```

Comment:
Do we need to show an error here? I'd assume that the condition is false, so we can silently do nothing.

Reply:
done